trp_use_case/view/use_case_collection.xml

```diff
@@ -41,7 +42,28 @@
                 </form>
             </field>
         </record>
-        <record id="use_case_collection_tree" model="ir.ui.view">
+
+	<record id="use_case_collection_search" model="ir.ui.view">
+            <field name="name">collection.search</field>
+            <field name="model">use_case.collection</field>
+            <field name="arch" type="xml">
+	      <search string="Search Status">
+
+		<filter name="draftstatus" string="Draft" icon="terp-stage"
+                        domain="[('x_status', '=', ['draft'])]"
```

Comment:
if you compare with an array, use the `'in'` operator. For the equality operator, use the scalar value